lib/opentuna-stack.ts

```diff
@@ -265,5 +266,12 @@ export class OpentunaStack extends cdk.Stack {
         ttl: cdk.Duration.minutes(5),
       });
     }
+
+    // invalidate cloudfront when web-portal changes
+    new CloudFrontInvalidate(this, 'CloudFrontInvalidate', {
+      distribution: distribution,
+      distributionPaths: ['/help', '/news', '/status', '/*.html'],
```

Comment:
pls also add `/` which is the default path. And the paths should be `/help/*`, `/news/*`, `/status/*`?